bindings/matrix-sdk-ffi/src/timeline/content.rs

```diff
@@ -189,8 +189,8 @@ impl Message {
 
 #[derive(uniffi::Record)]
 pub struct InReplyToDetails {
-    event_id: String,
-    event: RepliedToEventDetails,
+    pub(crate) event_id: String,
+    pub(crate) event: RepliedToEventDetails,
```

Comment:
Can it be a small constructor instead of two `pub(crate)` fields, please?

Reply:
sure! done!